cerbero/tools/sign.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# cerbero - a multi-platform build system for Open Source software
+# Copyright (C) 2019 Andoni Morales Alastruey <ylatuya@gmail.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+
+import os
+from cerbero.config import Platform
+from cerbero.utils import shell
+
+
+class Sign(object):
+    '''Wrapper for the sign tool'''
+
+    def __init__(self, config, package):
+        self.config = config
+        self.excludes = package.strip_excludes
+        self.package = package
+
+    def sign_file(self, path):
+        for f in self.excludes:
```

Comment:
Do you need to be greedy here? Meaning being `asd/esd` the file `f` and `this/is/asd/esd/isd/test.txt` the `path`, this will consider the file is excluded. In case we do not need to be greedy, we could simply use `if path in self.excludes` that will match only if any of the entries match exactly what's in `self.excludes`.